lib/samples/identify_raster_cell/identify_raster_cell.dart

```diff
@@ -0,0 +1,148 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_data.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+import 'package:path_provider/path_provider.dart';
+
+class IdentifyRasterCell extends StatefulWidget {
+  const IdentifyRasterCell({super.key});
+
+  @override
+  State<IdentifyRasterCell> createState() => _IdentifyRasterCellState();
+}
+
+class _IdentifyRasterCellState extends State<IdentifyRasterCell>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // Raster layer to display raster data on the map.
+  late RasterLayer _rasterLayer;
+  // Graphic to get the raster cell information.
+  final Graphic _textGraphic = Graphic();
+  // Graphics overlay to display the text graphic.
+  final _textGraphicsOverlay = GraphicsOverlay();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        left: false,
+        right: false,
+        child: Stack(
+          children: [
+            ArcGISMapView(
+              controllerProvider: () => _mapViewController,
+              onMapViewReady: onMapViewReady,
+              onTap: onTap,
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            LoadingIndicator(visible: !_ready),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future<void> onMapViewReady() async {
+    // Define a new map with Wgs84 Spatial Reference.
```

Comment:
I don't think this comment is quite accurate. I think we would normally just say,
```
  // Create a map with the oceans basemap style and an initial viewpoint.
```

Reply:
Actually, there's probably no reason to set the `initialViewpoint`, because the viewpoint will be changed immediately with the `_rasterLayer.fullExtent`. So we could just remove the `initialViewpoint` part and the above comment could just be "// Create a map with the oceans basemap style."